src/Tizen.NUI/src/public/Layouting/AbsoluteLayout.cs

```diff
@@ -54,6 +54,11 @@ protected override void OnMeasure(MeasureSpecification widthMeasureSpec, Measure
                     // Get size of child with no padding, no margin. we won't support margin, padding for AbsolutLayout.
                     MeasureChildWithoutPadding(childLayout, widthMeasureSpec, heightMeasureSpec);
 
+                    if(!childLayout.Owner.TransformedByLayout)
```

Comment:
I found there are a lot of `if(...)`(no space) in you patch here and I think it is because it is already used in `LinearLayout.cs`.
But I think it would be better to use `if (...)` at least to the newly added code.

Reply:
Thank you I changed it for all (except View.cs) patch codes